metrique/src/lib.rs

```diff
@@ -101,7 +101,7 @@ pub mod format {
 /// Test utilities for metrique
 #[cfg(feature = "test-util")]
 pub mod test_util {
-    pub use metrique_writer::test_util::{
```

Comment:
there is now the test_util contents of `writer` duplicated. thoughts?

Reply:
insufficient mood to remove it since that will cause annoying breakage